manywheel/build.sh

```diff
@@ -53,7 +53,7 @@ case ${CUDA_VERSION} in
         EXTRA_CAFFE2_CMAKE_FLAGS+=("-DATEN_NO_TEST=ON")
         ;;
     11.0)
-        export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;7.5;8.0"
+        export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;7.5;8.0;8.6"
```

Comment:
This would try to enable `sm86` on CUDA11.0 and should thus fail. 
The previous 11.1 block already has 8.6 in the architecture list.

Reply:
Correct, the question I have though is if CUDA 11.1 is actually required for `sm_86` or is it just that CUDNN 8.0.5 is required?

Reply:
CUDA11.1 is required. if you want to build for `sm_86`.

Reply:
Okay then, makes sense